liblouisutdml/logging.c

```diff
@@ -46,7 +46,7 @@ void EXPORT_CALL lbu_setLogLevel(logLevels level)
   logLevel = level;
 }
 
-void logMessage(logLevels level, const char *format, ...)
+void EXPORT_CALL /*lbu_*/logMessage(logLevels level, const char *format, ...)
```

Comment:
I would properly add the prefix here. If you add it to the API then let it have a prefix. I can do it if you like.

Reply:
I didn't add the prefix because I see it as an internal function really. It's just used in file2brl.

Reply:
OK, fair enough, but then shouldn't it be in `louisutdml.h` instead of `liblouisutdml.h`?

Reply:
I don't know. Would that work?

Reply:
Let me check

Reply:
Hm, cannot compile because my libslouis has `internal.h` removed, rebasing...

Seems to work, commit pending

Reply:
You tried cross compilation?

Reply:
Ah, blush, no

Reply:
Is there support for cross-compilation anywhere (Dockerfile, makefile, travis, anything)?

Reply:
Yes see Makefile.am. But it's probably out of date because it is not used a lot. I would like to make use of it in .travis/script/mingw.sh.

Reply:
Sorry can't find anything in the Makefile. I had a Dockerfile lying around (tbh don't know where that came from) but unfortunately it doesn't seem to be able to compile libxml2. It complains about 
``` console
threads.c:31:21: fatal error: windows.h: No such file or directory
 #include <windows.h>
                     ^
compilation terminated.
```

Reply:
Tried unsuccessfully to attach it to the conversation, so just pushed

Reply:
What do you mean you can't find anything in the Makefile. It's the only thing that's in it. And it uses the Dockerfile inside Dockerfile.dev/.

Reply:
Oh you mean in Liblouisutdml. Right. I use liblouisutdml-nar for that. But it has local changes which I can't push yet.

Reply:
Maybe it's easier if I test it.

Reply:
Commit 2e4c9f58d90f9f97179cc86d0a8da0d2f7c3d599 breaks it for me.

Reply:
In what way? How does it break?

Reply:
Just doesn't build anymore when cross-compiling. Same behavior as before my commit https://github.com/liblouis/liblouisutdml/pull/57/commits/6a001b1700cf666e41a321e576301c190ba227d1.